tools/src/main/java/org/apache/rocketmq/tools/command/export/ExportMetadataCommand.java

```diff
@@ -165,11 +167,10 @@ public void execute(CommandLine commandLine, Options options, RPCHook rpcHook)
                         result.put("rocketmqVersion", MQVersion.getVersionDesc(MQVersion.CURRENT_VERSION));
                         result.put("exportTime", System.currentTimeMillis());
 
-                        filePath = filePath + "/metadata.json";
```

Comment:
note that topicConfigMap is not new for current broker, it accumulates the previous broker queue numbers.